spec/models/returning_teacher_spec.rb

```diff
@@ -13,6 +13,20 @@
     end
   end
 
+  describe "#set_degree_options" do
+    it "sets the degree option before validation" do
+      returning_teacher.valid?
```

Comment:
Shouldn't need these `valid?` calls any more as it's set as a default on the attribute

Reply:
👍